### PR TITLE
Add Aniyomi

### PR DESCRIPTION
**Pull request by OLuwis**
- Aniyomi is a Tachiyomi fork that enables streaming features for anime watching.
- Mihon is a Tachiyomi successor from the developer of another fork called [TachiyomiSY](https://www.reddit.com/r/animepiracy/comments/197vwou/first_version_of_mihon_a_tachiyomi_succesor_from/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,8 +117,10 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • Anime & Manga
 
+* [**Aniyomi**](https://github.com/aniyomiorg/aniyomi)
 * [**Bangumi**](https://github.com/czy0729/Bangumi) <sup>**[[F-Droid](https://www.f-droid.org/app/com.czy0729.bangumi)]**</sup>
 * [**Kotatsu**](https://github.com/nv95/Kotatsu) <sup>**[[F-Droid](https://f-droid.org/app/org.koitharu.kotatsu)]**</sup>
+* [**Mihon**](https://github.com/mihonapp/mihon)
```
Comment (offa) [inline README.md:123]: That one seems to include proprietary Google components.

Comment (OLuwis) [reply README.md:123]: Yeah, you are correct, they do use Firebase for crash analytics/reports.
You can check it [here](https://github.com/mihonapp/mihon/blob/main/app/build.gradle.kts).

Comment (offa) [reply README.md:123]: Mihon doesn't meet the inclusion criteria, but Aniyomi seems fine.

Comment (OLuwis) [reply README.md]: Done.

**Comment by offa** [comment]
Thank you! :-)

**Comment by shuvashish76** [comment]
- https://github.com/aniyomiorg/aniyomi/issues/60
- https://github.com/mihonapp/mihon/issues/1 (not planned for F-Droid)

**Pull request merged**